content/applications/finance/payment_acquirers/adyen/update.rst

```diff
@@ -0,0 +1,54 @@
+=================================
+Install the Adyen PayByLink Patch
+=================================
+
+Starting October 2022, due to a change in their API, :doc:`Adyen <../adyen>` needs to be updated to
```

Comment:
Rather than evoking "a change in their API" (not very clear), let's directly link readers to the official announcement: https://docs.adyen.com/online-payments/classic-integrations/hosted-payment-pages/deprecation